pkg/kubectl/describe.go

```diff
@@ -842,21 +842,25 @@ func describeContainers(label string, containers []api.Container, containerStatu
 		if len(resourceToQoS) > 0 {
 			fmt.Fprintf(out, "    QoS Tier:\n")
 		}
-		for resource, qos := range resourceToQoS {
+		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
```

Comment:
Qos has changed this release to be at the pod level and not for individual
compute resources.  A follow on PR was made that now reports the effective
QoS for a pod in describe.

On Thursday, May 26, 2016, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/kubectl/describe.go
> <https://github.com/kubernetes/kubernetes/pull/26046#discussion_r64710897>
> :
>
> > @@ -842,21 +842,25 @@ func describeContainers(label string, containers []api.Container, containerStatu
> >  		if len(resourceToQoS) > 0 {
> >  			fmt.Fprintf(out, "    QoS Tier:\n")
> >  		}
> > -		for resource, qos := range resourceToQoS {
> > +		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
>
> I think we should stop reporting qos per resource in 1.3
>
> Does this mean that if I have a deployment and describe it I won't be able
> to see its QoS anymore?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26046/files/c8a14e376e3df237d6d0d6a2d5266056cbfeab87#r64710897>
>